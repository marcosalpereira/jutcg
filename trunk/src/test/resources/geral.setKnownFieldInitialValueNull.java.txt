package br.gov.esaf.sgc.qqpacote;

public class Bean implements CampoAtuacaoBusiness {
    private Var field;

    public void test() {
        if(field == null) {
        	System.out.println("true");
        }
    }    
}