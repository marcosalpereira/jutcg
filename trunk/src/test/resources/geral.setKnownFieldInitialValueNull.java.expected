package br.gov.esaf.sgc.business.impl;

import static org.easymock.EasyMock.*;
import org.junit.*;
import java.util.*;
import static org.junit.Assert.*;

/**
 * Testes unitarios para a classe {@link Bean}.
 */
public class TestBean  {

/**
 * Classe sendo testada {@link Bean}.
 */
private Bean bean;

/**
 * Configuracoes iniciais.
 */
@Before
public void setUp() {
  bean = new Bean();
}

/**
 * Teste para o metodo {@link Bean#test()}.
 * Descricao do Fluxo: 
 *     Entra: if(field == null) {.
 */
@Test
public void testTestPath1() {

// variaveis usadas
final Var field = null;

// Configurando estado interno da classe sob teste
bean.setField(field);

// invocar metodo sendo testado
bean.test();
}

/**
 * Teste para o metodo {@link Bean#test()}.
 * Descricao do Fluxo: 
 * NAO Entra: if(field == null) {.
 */
@Test
public void testTestPath2() {

// variaveis usadas
final Var field = new Var(1);

// Configurando estado interno da classe sob teste
bean.setField(field);

// invocar metodo sendo testado
bean.test();
}
}
